src/pages/Login/LoginForm.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+//styles
+import classNames from 'classnames/bind';
+import styles from './LoginPage.css'
+import 'coupon-components/build/styles.css';
+//components
+import { InputBox, Button, Typography, Card } from 'coupon-components';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+import Logo from 'Components/Logo/Logo';
+
+const cx = classNames.bind(styles);
+
+class LoginForm extends React.Component {
+  state = {
+    credentials: {
+      email: '',
+      password: ''
+    }
+  }
+
+  onChange = (event) => {
+    this.setState({
```

Comment:
Si ya estamos usando redux form ya come todas las variables que tenga en el onSubmit(values) no es necesario este this.setState.